### ISSUE TITLE
Comments in quasiquotes

### ISSUE THREAD

**Issue by MasseGuillaume** [scalameta/scalameta#46]
``` scala
q"""
object $name {
$unquotee // single line
/* multi
   lines */
"""
```

@densh pretty please :)

**Comment by xeno-by** [comment]
Here's an unfinished attempt at fixing this issue: https://github.com/scalameta/scalameta/pull/510.

**Comment by xeno-by** [comment]
@olafurpg and I had a discussion, and we decided to pronounce quasiquotes feature-complete.

There are still some open issues (#46, #223, #251, #280 and others), but they all have two things in common: 1) they require non-trivial design work to be fixed, 2) they aren't popular at all. Given our recent decision to start avoiding unwarranted breaking changes in 2.0, we'll consider quasiquotes good enough at this point and will close all outstanding feature requests.

We will still be fixing bugs (in fact, I've fixed about a dozen of such bugs for 2.0), but without strong evidence our design efforts will be directed elsewhere.
